app/views/shared/_user_profile.html.erb

```diff
@@ -0,0 +1,9 @@
+<li>
+  <div class="user_img">
+    <img src="<%= user.Photo %>" alt="Photo">
+  </div>
+  <div class="user_details">
+    <h2><%= user.Name %></h2>
+    <p>Number of Posts: <%= user.Posts_counter.nil? ? '0' : user.Posts_counter %></p>
+  </div>
```

Comment:
- [ ] Great job on your implementation so far! 🚀 However, there are a few minor issues that need some attention to ensure your code functions as expected. 

   - It appears that the data from the database isn't displaying correctly in the UI. This issue arises from the way you're referencing your data column names. I've noticed that you're using capital letters at the beginning of the column names, but they should match how they are saved in the database, typically with lowercase letters. 

   For example, in the highlighted code, you can access the `photo` from your database like this:

   ```ruby
   <img src="<%= user.photo %>" alt="Photo">
   ``` 

   Instead of:

   ```ruby
   <img src="<%= user.Photo %>" alt="Photo">
   ``` 

   - Kindly make sure to make this correction across your entire codebase. Some of the column names you need to correct in this file includes: `photo`, `name`, and `posts_counter`.

Happy coding! 😊👨‍💻🚀